frontend/src/component/operator-wizard/OperatorWizard.js

```diff
@@ -319,51 +319,55 @@ export default class OperatorWizard extends Component {
           page.subPages.length > 0
         ) {
           page.subPages.forEach(subPage => {
-            if (
-              (subPage.label === SMART_ROUTER_STEP ||
-                subPage.label === PIM_STEP) &&
-              this.stepBuilder.getObjectMap(ENV_KEY).startsWith(RHDM_ENV_PREFIX)
-            ) {
-              //do not add in yaml
-            } else {
-              let subPageFields = subPage.fields;
+            if (subPage.visible !== undefined && subPage.visible === true) {
+              if (
+                (subPage.label === SMART_ROUTER_STEP ||
+                  subPage.label === PIM_STEP) &&
+                this.stepBuilder
+                  .getObjectMap(ENV_KEY)
+                  .startsWith(RHDM_ENV_PREFIX)
+              ) {
+                //do not add in yaml
```

Comment:
Is it just a empty {} here?

Reply:
It is existing code, was added earlier to skip yaml generation for smartrouter and pim for RHDM environments.